saltstack/final/salt/openscap/init.sls

```diff
@@ -0,0 +1,19 @@
+openscap_install:
```

Comment:
A few general comments for this file:

1. I think this should only run when stig is enabled
2. The installed packages should be cleaned up, similarly to ansible in cis-controls
3. I guess the results should be under /tmp/cis too, so that we can see it on Jenkins side

Reply:
In the top.sls the stig_enebled var is checked.
I do not think that the openscap install log have to be collected for every image. If install issue occurs during burning time it can be investigated based on the jenkins log.
Maybe the openscap can be useful for newer checks in the future for example during engesc...